core/src/main/scala/org/apache/spark/api/python/PythonRunner.scala

```diff
@@ -59,6 +59,9 @@ private[spark] object PythonEvalType {
   val SQL_TABLE_UDF = 300
   val SQL_ARROW_TABLE_UDF = 301
 
+  val SQL_STREAMING_FOREACH_BATCH = 400
+  val SQL_STREAMING_LISTENER = 401
```

Comment:
can we have a separate worker file instead of using eval? event handling is async so it might conflict with other existing running Python workers. At least they would affect each other on execution time I susepct?